BuildingSystems/Applications/DistrictSimulation/HCBC.mo

```diff
@@ -7,6 +7,7 @@ model HCBC
   model Building
     "Building model with four capacities used in conjunction with parameter maps for campus simulations"
   extends BuildingSystems.Buildings.BaseClasses.BuildingTemplate(
+    nIdealLoads = 1,
```

Comment:
@Zohra-D what does the value mean ? could you document it ?
I see 1 2 and 13 as values

Reply:
`nIdealLoads` corresponds to the number of thermal zones for which we calculate, at each time step, the right heat or cooling flow to maintain a comfort temperature inside the zone.

Following the definitions of calcIdealLoads and nIdealLoads in BuildingSystems : 

- calcIdealLoads : 
```
parameter Boolean calcIdealLoads = true
  "True: calculation of the ideal heating and cooling load; false: no calculation"
  annotation(HideResult = true,Dialog(tab="General",group="Heat and cooling load calculation"));
```

- nIdealLoads : 
```
  parameter Integer nIdealLoads = 0
    "Number of thermal zones of the building where ideal loads are calculated"
    annotation(Evaluate=true, Dialog(connectorSizing=true, tab="General",group="Ports"));
```